docs/manuals/source/manually_added_config_directive_descriptions/dir-director-NdmpNamelistFhinfoSetZeroForInvalidUquad.rst.inc

```diff
@@ -0,0 +1 @@
+This directive enables a bug workaround for Isilon 9.1.0.0 systems where the NDMP namelists tape offset (also known as fhinfo) is sanity checked so that the valid value -1 is not accepted anymore. The Isilon system sends the following error message: 'Invalid nlist.tape_offset -1 at index 1 - tape offset not aligned at 512B boundary'. The workaround sends 0 instead of -1 which is accepted by the Isilon system and enables normal operation again.
```

Comment:
```suggestion
This directive enables a bug workaround for Isilon 9.1.0.0 systems where the NDMP namelists tape offset (also known as fhinfo) is sanity checked resulting in valid value -1 being no more accepted. The Isilon system sends the following error message: 'Invalid nlist.tape_offset -1 at index 1 - tape offset not aligned at 512B boundary'. The workaround sends 0 instead of -1 which is accepted by the Isilon system and enables normal operation again.
```